# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../firefox/cargo.patch
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-67.0/build/moz.configure/rust.configure.vanilla	2019-05-25 23:55:47.372465403 +0200
+++ firefox-67.0/build/moz.configure/rust.configure	2019-05-25 23:56:36.268464412 +0200
@@ -71,7 +71,7 @@
         cargo_min_version = Version('0.23.0')
     else:
         rustc_min_version = Version('1.32.0')
-        cargo_min_version = rustc_min_version
+        cargo_min_version = Version('0.23.0')
 
     version = rustc_info.version
     if version < rustc_min_version:
